src/quo2/components/list_items/preview_list.cljs

```diff
@@ -0,0 +1,100 @@
+(ns quo2.components.list-items.preview-list
+  (:require [quo.react-native :as rn]
+            [status-im.i18n.i18n :as i18n]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as quo2.icons]
+            [quo2.components.markdown.text :as quo2.text]))
+
+(def params {32 {:border-radius {:circular 16
+                                 :rounded  10}
+                 :hole-radius   {:circular 18
+                                 :rounded  12}
+                 :margin-left   -8
+                 :hole-size     36
+                 :hole-x        22
+                 :hole-y        -2}
```

Comment:
Maybe separate params in different objects so it is easier to read

Reply:
Improved params, so they are more readable now
![3](https://user-images.githubusercontent.com/17097240/189546452-94bb9701-8f29-41c1-ae01-925e8c4a344b.png)